src/logger.js

```diff
@@ -0,0 +1,35 @@
+const
```

Comment:
This isn't really a review on the code, but in general it might be helpful to consider the "reporter" approach that things like [gulp](https://github.com/gucong3000/gulp-reporter) and [mocha](https://mochajs.org/#reporters) use, so the tool could be used more flexibly.